internal/db/schema/migrations/postgres/2/01_credential.down.sql

```diff
@@ -0,0 +1,14 @@
+begin;
```

Comment:
I thought we didn't do down files anymore?

Reply:
They can be useful during development so they are allowed on branches.